### ISSUE TITLE
[A11y] Desktop/Mobile Collection Landing: Breadcrumbs Active Page

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1746]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

- [ ] Yes

### Issue Description

Collection Landing Page: https://staging.oregondigital.org/collections/gwilliams?locale=en

When I tab through the page and land on the breadcrumbs it will tab and read "Home" but skips and doesn't read "Geralds W. Williams Collection" the active page I am on.

### Suggested Fix

I mentioned in a previous accessibility ticket #1745 for the possible fix.

a element needs the following:
aria-current=page

See w3c recommendation:
https://w3c.github.io/aria-practices/examples/breadcrumb/index.html
